utils/utils.go

```diff
@@ -263,6 +264,15 @@ func CreateClient(conf NetConf) (*client.Client, error) {
 	return calicoClient, nil
 }
 
+func IsReady(client *client.Client) bool {
+	kvPair, err := client.Backend.Get(model.ReadyFlagKey{})
+	if err != nil {
+		return true
+	}
+
+	return kvPair.Value.(bool)
```

Comment:
I think we should check the interface assertion, because someone can manually set the value to something and all the CNI plugins will start panicing.